contrib/docs/tutorials/authenticate-api-requests.md

```diff
@@ -6,6 +6,13 @@ API requests from frontend plugins include an authorization header with a Backst
 
 **NOTE**: Enabling this means that Backstage will stop working for guests, as no token is issued for them.
 
+Since the middleware always expects a valid token, API requests from backend plugins will need one as well. Backends have no concept of a Backstage identity so instead they use a token generated using a shared key from the `app-config.yaml`.
```

Comment:
I'd really like to see us take ownership of this documentation into the main docs site. Linking out to `contrib` docs doesn't really feel great, as they're unstable and can quickly move out of date. Maybe it's not the right time to do this yet. But I think it would be good to note that we are now supporting this stuff as a first class thing.

Reply:
maybe it could be moved into `docs/tutorials` for the time being? i imagine when the authorization framework is fully shipped we'll be adding a dedicated section for that in the docs and either removing this tutorial entirely or maybe moving it there and flagging it as the outdated approach.

happy to add a note at the top here as well, though, to indicate that this approach now works end-to-end and maybe that we're actively iterating on it? any suggestion on the wording?

Reply:
moved and added a note at the top!